tests/test_fp8_quantize.py

```diff
@@ -2,6 +2,8 @@
 import torch
 
 from flashinfer import mxfp8_dequantize_host, mxfp8_quantize
+from parameterized import parameterized
```

Comment:
Can we rely on `pytest.mark.parametrized` instead?
Or adding `parameterized` to docker container: https://github.com/flashinfer-ai/flashinfer/blob/main/docker/install/install_python_packages.sh

Reply:
updated